boilerplate/e2e/README.md

```diff
@@ -0,0 +1,66 @@
+# Detox End-To-End Testing
+
+## Setup
+
+To get your Detox tests up and running, you'll need to install some global dependencies:
+
+1. Install the latest version of [Homebrew](https://brew.sh/)
```

Comment:
Does this only work on macOS? We should mention that above, probably. Maybe we should also check if they're on macOS before even asking if they want Detox. Could put something like this:

```
Skipping Detox because it is only supported on macOS and you're on Windows...
```

Reply:
Yes, the tests require XCode (Detox is only stable on iOS at the moment), so that's a good point.

Reply:
Ah, I see https://github.com/infinitered/ignite/blob/master/src/commands/doctor.js#L6